assets/js/blocks/cart-checkout/mini-cart-contents/editor.scss

```diff
@@ -1,13 +1,13 @@
 .wp-block-woocommerce-mini-cart-contents {
 	max-width: 480px;
-	margin: 0 auto;
+	margin: 0 auto !important;
```

Comment:
Maybe is it better to add a comment that explains why we are using `!important`

Reply:
Actually we don't need this with changes introduced in 5d6818e

Reply:
Turned out that we need it. I added the comment to clarify the reason.

Reply:
Mmm, are you sure about this?

In my local env, I tried and I see a different result with and without `!important`

![image](https://user-images.githubusercontent.com/4463174/147077012-c9a809e2-7187-4fd5-a2af-87062b414762.png)
* with !important

![image](https://user-images.githubusercontent.com/4463174/147077149-0888e971-cb27-497c-8282-a53d18649f4d.png)
* without !important

Check the scrollbar on the right

Reply:
Oh, I see that you noticed the difference and add a comment. For me, now it's perfect 🚢